src/audio/component.c

```diff
@@ -136,8 +136,6 @@ int comp_set_state(struct comp_dev *dev, int cmd)
 
 	dev->state = requested_state;
 
-	comp_writeback(dev);
```

Comment:
Minor typo in this commit message (coferent -> coherent)

Commit title: "cache: remove wrong component cache synchronisation"